app/Services/PgBackRestService.php

```
@@ -0,0 +1,474 @@
+    }
+
+    /**
+     * Perform a backup with pgBackRest
+     */
+    public function performBackup(string $type = 'incr'): array
+    {
+        Log::info('Starting pgBackRest backup', [
+            'database' => $this->database->uuid,
+            'type' => $type,
+            'stanza' => $this->stanzaName,
+        ]);
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Support\Facades\Log' in method 'performBackup'. (undefined)

(StaticAccess)